Source/DtoImporterService/CategoryImporter.cs

```diff
@@ -65,6 +65,8 @@ private int upsertCategories(List<Ladder[]> ladders)
 					var id = ladder[i].CategoryId;
 					var name = ladder[i].CategoryName;
 
+					if (name == "Storia") ;
```

Comment:
Aside from the fact that I don't know what "Storia" is, this looks like an unintentional way to declare an empty statement.

Reply:
Sorry, I pushed these to save my work remotely while I fugue out what the hell is going on here. I'm going to clean it up before I ask for a review.

It turns out that trying to modify Book.CategoryLink starts another tracking instance, so the context can't be saved. I _thin_ it has to do with EF's automatic configuration of the many-to-many, but really I'm just fumbling around here trying to sort this out.